proposals/hackygolucky-EXPECTATIONS/EXPECTATIONS.md

```diff
@@ -0,0 +1,108 @@
+## Expectations
+
+With the intent to merge the Node.js Foundation and JS Foundation being announced, those interested in the continued support of Node.js and JS-related projects are working through expectations and structure for what the potential new foundation could look like.
+
+This document is for the initial phase of work for the Bootstrap team to draft requirements that overlap with scope of the Board in order to provide an outline and structure for the foundation attuned to the needs of the projects and ecosystem stakeholders who have participated.
+
+This document is organized by two key areas: concerns that must be included in the foundation bylaws, and those that are communicated expectations that aren't required to be in our legal documents to proceed but are vital to managing expectations of project members and ecosystem stakeholders.
+
+### Bylaws requirements
+- Communicated scope for the foundation and board 
+- Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement 
+- The foundation staff, executive director, and board should follow a leadership style that prioritizes and promotes the well-being of the project and those who prioritize and promote a healthy project and ecosystem. 
+- Project representation on the Board 
+    - Community Representation should sit on the Board directly. Projects could share this representation. 
+    - Community representation on the Board: 3 seats guaranteed for community representation 
+- Board must establish Cross Project Council focused on the specific needs of the member projects 
+    - Allow projects to self-organize amongst themselves. Processes are defined and driven by project e.g. CPC is not structured by the Board 
+- Non-corporate member representation in budget/finance subcommittee 
+- Projects values should drive foundation focus and priority 
+- No "pay to play" in technical participation 
+- Earmarking allowed with flexibility in budgeting for concerns and needs of individual projects 
```

Comment:
We may want to expand this along these lines:

The ability for foundation members to direct a portion of their contributions to a specific project. As one example there could be a base fee to join the foundation which is shared across the project and then the option to join as a specific member for a particular project with an additional cost.  Possibly capped by your membership level to the overall foundation.  So for example, if you are a gold member of the overall foundation, then you can become a Node.js gold member as well but not a Node.js platinum member.   We've heard companies say that they may be interested in directing their funds so we should make sure we include options to allow for that.

Reply:
I explicitly left this vague because this is a thing that _really_ should be written by lawyers and finance folks. 

Are you saying, additionally, you think we should allow for 'membership' in a single project? Directing funds is one thing to single projects in a foundation, but membership to a single project in an overarching foundation is very differnent(membership is actually part of legal structure separate from what we call members of the org in GitHub) 

I don't think we've captured that anywhere but I'm wondering what that would look like. I'm not sure I've seen a model for this in a real world scenario yet...

Reply:
The model I've heard used by other foundations is that first you join the overall foundation at cost X.  Then if you want to additionally support a specific project you "join" that specific project as well at cost Y.  So you end up paying X which goes to the overall foundation/split across projects and Y which goes specifically to the project.

Reply:
Right, so I think my drawn out comment there was to say: I haven't seen documented conversations about this type of structure yet with project(s) folks, with members of any Board, or the LF, so I think this is yet another thing we need to break out as a separate issue to flush out the details. This isn't a small thing and we'd want to make sure most/all of the projects are on board with this idea.